clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -864,7 +867,7 @@ public boolean exists(Path path) throws IOException {
         // number of objects so that it costs about the same to list that
         // many objects as it does to list 1.
         try {
-            ObjectStatsList osl = objects.listObjects(objectLoc.getRepository(), objectLoc.getRef(), false, false, "", 123 /* TODO(ariels): configure! */, "", objectLoc.getPath());
+            ObjectStatsList osl = objects.listObjects(objectLoc.getRepository(), objectLoc.getRef(), false, false, "", 5, "", objectLoc.getPath());
```

Comment:
Understood why it will help. Will be good to have @arielshaqed to review this one too.

Reply:
I agree that it will help... in _some_ instances.  Let's make it a configuration?  We _will_ have someone writing `.../path/to/parquet/...` who already has `.../path/to/parquet.orig/...` -- and having just 5 will hit them because "`.`" is before "`/`".

Or if we don't make it configurable, please keep the TODO.